build/azure-pipeline/workflow-pr.yml

```diff
@@ -172,11 +156,8 @@ stages:
           #   displayName: Build C Examples
           
           #
-          # Running all tests
+          # Running integration tests
           #
-          - script: make test
```

Comment:
some tests might use environment variables, I don't think we should remove tests (e.g., #153)

Reply:
we have github action dedicated to running unittests, so I believe it's okay to not run them again in our integration pipelines.